Source/ACE.Server/WorldObjects/Creature_Death.cs

```diff
@@ -223,10 +223,8 @@ protected void CreateCorpse(WorldObject killer)
             }
             else
                 corpse.LongDesc = $"Killed by misadventure.";
-
-            bool saveCorpse = false;
```

Comment:
This pr is great!

Looks like the force corpse save code was not merged properly though.